packages/components/Notification/src/__tests__/__snapshots__/Notification.test.tsx.snap

```diff
@@ -14,221 +24,148 @@ exports[`Notification component tests Notification default 1`] = `
       },
       "borderColor": "transparent",
       "borderRadius": 12,
-      "bottom": undefined,
-      "end": undefined,
+      "borderWidth": 1,
       "flex": 1,
+      "flexDirection": "row",
       "justifyContent": "space-between",
-      "left": undefined,
-      "margin": undefined,
-      "marginBottom": undefined,
-      "marginEnd": undefined,
       "marginHorizontal": 16,
-      "marginLeft": undefined,
-      "marginRight": undefined,
-      "marginStart": undefined,
-      "marginTop": undefined,
-      "marginVertical": undefined,
       "minHeight": 52,
-      "right": undefined,
-      "shadowColor": "#000000",
-      "shadowOffset": Object {
-        "height": 8,
-        "width": 0,
-      },
-      "shadowOpacity": 0.14,
```

Comment:
Should we be worried that the new snapshots don't include anything about shadows?

Reply:
Oh... is this a "shadows aren't defined on default theme" kinda thing? ._.

Reply:
Thanks for noticing that @mischreiber  - hmm I think they should be defined on the default theme? Otherwise none of the snapshots in any of the tests would have any shadows. I'll first try to fix the PR checks to see if that changes anything, and then reevaluate whether this is an issue or not

Reply:
Fixed now, was trying to access t.colors.shadow16 instead of t.shadows.shadow16...colors have been getting to my head. Snapshots make a lot more sense now